schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/ISO19139/mapping/core.xsl

```diff
@@ -231,18 +231,16 @@
   </xsl:template>
   <xsl:template match="gmd:metadataStandardName" priority="5" mode="from19139to19115-3.2018">
     <!--
-      metadataStandardName and gmd:metadataStandardVersion are combined into a CI_Citation
+      Output the converted record's new metadata standard name
     -->
     <mdb:metadataStandard>
       <cit:CI_Citation>
-        <xsl:call-template name="writeCharacterStringElement">
-          <xsl:with-param name="elementName" select="'cit:title'"/>
-          <xsl:with-param name="nodeWithStringToWrite" select="."/>
-        </xsl:call-template>
-        <xsl:call-template name="writeCharacterStringElement">
-          <xsl:with-param name="elementName" select="'cit:edition'"/>
-          <xsl:with-param name="nodeWithStringToWrite" select="../gmd:metadataStandardVersion"/>
-        </xsl:call-template>
+        <cit:title>
+          <gco:CharacterString>ISO 19115-3:2018</gco:CharacterString>
+        </cit:title>
+        <cit:edition>
+          <gco:CharacterString>1.0</gco:CharacterString>
+        </cit:edition>
       </cit:CI_Citation>
     </mdb:metadataStandard>
   </xsl:template>
```

Comment:
It has been discussed here https://gitter.im/geonetwork/core-geonetwork?at=622fd30c8db2b95f0a697bfd. Preserving custom values (like profile details) are still relevant I think, so I would only set hardcoded value only for common case.

eg.
```xml
  <!-- Replace default standard name to fixed value ....-->
  <xsl:template match="gmd:metadataStandardName[
                          gcoold:CharacterString = ('ISO19115', 'ISO19139', 'ISO 19115:2003/19139')]"
                priority="5" mode="from19139to19115-3.2018">
    <mdb:metadataStandard>
      <cit:CI_Citation>
        <cit:title>
          <gco:CharacterString>ISO 19115-3:2018</gco:CharacterString>
        </cit:title>
        <cit:edition>
          <gco:CharacterString>1.0</gco:CharacterString>
        </cit:edition>
      </cit:CI_Citation>
    </mdb:metadataStandard>
  </xsl:template>

  <!--
    or combined custom ones metadataStandardName and gmd:metadataStandardVersion
    into a CI_Citation
  -->
  <xsl:template match="gmd:metadataStandardName" priority="5"
                mode="from19139to19115-3.2018">
    <mdb:metadataStandard>
      <cit:CI_Citation>
        <xsl:call-template name="writeCharacterStringElement">
          <xsl:with-param name="elementName" select="'cit:title'"/>
          <xsl:with-param name="nodeWithStringToWrite" select="."/>
        </xsl:call-template>
        <xsl:call-template name="writeCharacterStringElement">
          <xsl:with-param name="elementName" select="'cit:edition'"/>
          <xsl:with-param name="nodeWithStringToWrite" select="../gmd:metadataStandardVersion"/>
        </xsl:call-template>
      </cit:CI_Citation>
    </mdb:metadataStandard>
  </xsl:template>
```


> Without this fix, when there are a lot of records to upgrade, it is very laborious. Every record has to be edited after conversion.

Check batch editing maybe ? https://geonetwork-opensource.org/manuals/4.0.x/en/user-guide/workflow/batchediting.html

Reply:
Hi Francois.  I think this is a bug that ought to be fixed. I have implemented your suggested fix.